openstack/cinder/templates/etc/_cinder.conf.tpl

```diff
@@ -106,3 +106,6 @@ region_name = {{.Values.global.region}}
 
 [coordination]
 backend_url = memcached://{{ .Chart.Name }}-memcached.{{ include "svc_fqdn" . }}:{{ .Values.memcached.memcached.port | default 11211 }}
+
+[nova]
+interface=admin
```

Comment:
We seem to use spaces around `=` in other places. We should keep the format the same through the file if possible.